src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/client/prisonapi/PrisonMovementEvents.kt

```diff
@@ -11,5 +11,5 @@ package uk.gov.justice.digital.hmpps.educationandworkplanapi.app.client.prisonap
  */
 data class PrisonMovementEvents(
   val prisonNumber: String,
-  val prisonMovements: Map<Long, List<PrisonMovementEvent>>,
+  val prisonBookings: Map<Long, List<PrisonMovementEvent>>,
```

Comment:
I think this is a better name, since the map is keyed by prison "bookings". Also, it was easy to get lost in amongst terms such as `prisonMovements` and `prisonMovementEvents` etc, so this makes more sense to me.